### PR TITLE
Method to check if session has audio

### PR DESCRIPTION
**Pull request by SebDuf**

### CODE REVIEW

File: src/web-rtc-client.js [hunk 1]
```diff
@@ -491,6 +491,30 @@ export default class WebRTCClient extends Emitter {
...
   }
 
+  sessionHasAudio(session: Session) {
+    const pc = session.sessionDescriptionHandler.peerConnection;
+    let hasAudio = false;
+    if (pc.getSenders) {
+      pc.getSenders().forEach(sender => {
+        if (sender.track && sender.track.kind === 'audio') {
+          if (sender.track.enabled) {
+            hasAudio = true;
```
Comment (manuquentin) [inline src/web-rtc-client.js]: You can `return true` and `return false` at the end of the method. It will save you a variable

Comment (SebDuf) [reply src/web-rtc-client.js]: A `return` inside a `forEach` only ends the current iteration, the function won't receive the returned value. The forEach creates a new scope which captures the return.

https://www.competa.com/blog/the-javascript-array-foreach-method-doesnt-return-anything/

Comment (manuquentin) [reply src/web-rtc-client.js]: Oh sorry, I didn't see the `forEach`, can it be a `for( ... of ... ) ` ?

Comment (SebDuf) [reply src/web-rtc-client.js]: A return inside a `for ... of ...` interrupts the iterator.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Closing_iterators

**Pull request merged**